erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -267,7 +258,7 @@ namespace erizo {
 
   void WebRtcConnection::onCandidate(const CandidateInfo& cand, Transport *transport) {
     std::string sdp = localSdp_.addCandidate(cand);
-    ELOG_DEBUG("On Candidate %s", sdp.c_str());
+    ELOG_DEBUG("%s, message: Discovered New Candidate, Candidate: %s", toLog(), sdp.c_str());
```

Comment:
I think your idea is to use camelCase for field names, if so use `..., candidate: ...`